### ISSUE TITLE
Why would anyone would want to fork?

### ISSUE THREAD

**Issue by StefanGreve** [pascalandy/GNU-GENERAL-PUBLIC-LICENSE#2]
I really appreciate the sentiment and intention of this repository, but following your chain of arguments, why would anyone would want to fork your repository in order to obtain the license text of the GNU General Public License? As far as I am aware, there already exists an official markdown version of this license on the [GNU website](http://www.gnu.org/licenses/gpl-3.0.html). They even provide various other files ready to download, inter alia LaTeX, HTML, ODF, and RTF. In fact, 

> If any changes happen, how do you know it's still right for you?

I'll throw this question right back at you ;) From my point of view, you're a third-party provider for a license text that's already publicly available in markdown.

**Comment by pascalandy** [comment]
> From my point of view, you're a third-party provider for a license text that's already publicly available in markdown.

Correct! The point is to create awareness about forking a licence so **you know it won't change**. The last thing to do is to point back your license to this repo. Make sense?

**Comment by pascalandy** [comment]
> From my point of view, you're a third-party provider for a license text that's already publicly available in markdown.

Correct! The point is to create awareness about forking a licence so you know it won't change. The last thing to do is to point back your license to this repo. Make sense?

**Comment by StefanGreve** [comment]
Yes, and I am glad that you clarified the point you are trying to make.

**Comment by pascalandy** [comment]
Cool, I was about to update the readme but Github is not taking my commits at the moment :-/
https://www.githubstatus.com/

Cheers!
